modules/microshift-etcd-version.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * microshift_troubleshooting/microshift-version.adoc
+// * microshift_support/microshift-etcd.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-version-etcd_{context}"]
+= Checking the etcd version
+
+To enhance troubleshooting, you can get the version information for the etcd database included with your {microshift-short} version.
```

Comment:
I'm not sure about this sentence. The bottom line is that we should tell users - here's how you get etcd version.